app/src/androidTest/java/org/mozilla/fenix/ui/robots/LibraryRobot.kt

```diff
@@ -34,13 +39,57 @@ class LibraryRobot {
             HomeScreenRobot().interact()
             return HomeScreenRobot.Transition()
         }
+
+        fun closeMenu(interact: HomeScreenRobot.() -> Unit): HomeScreenRobot.Transition {
+            mDevice.waitForIdle()
+            closeButton().click()
+
+            HomeScreenRobot().interact()
+            return HomeScreenRobot.Transition()
+        }
+
+        fun openBookmarks(interact: BookmarksRobot.() -> Unit): BookmarksRobot.Transition{
+            mDevice.waitForIdle()
+            bookmarksButton().perform(click())
+
+            BookmarksRobot().interact()
+            return BookmarksRobot.Transition()
+        }
+
+        fun openHistory(interact: HistoryRobot.() -> Unit): HistoryRobot.Transition{
+            mDevice.waitForIdle()
```

Comment:
same, see:  https://github.com/mozilla-mobile/fenix/pull/4770/files#r316397535
but only add these waits if necessary (i.e. if test fails with an element not found)